### ISSUE TITLE
[BUG] Query broken by 451 PR for PostgreSQL

### ISSUE THREAD

**Issue by im-kulikov** [75team/thinkjs#459]
## DESC

@welefen and @GuilhermeReda this PR (#451) break queries.

### Explain:
```js
yield this.model("some_table")
          .field([
            '"some_table"."some_field_1" AS "field_1"',
            '"some_table"."some_field_2" AS "field_2"',
            '"some_table"."some_field_3" AS "field_3"',
            ...
            '"another_table"."some_field_N" AS "field_N"',
          ])
          .join('"another_table" ON (....)')
          .where(`"some_table"."some_field_1" = ${some_var}`)
          .select();
```

this will generate in PostgreSQL next query:

```sql
select ""some_table"."some_field_1" AS "field_1"",
            ""some_table"."some_field_2" AS "field_2"",
            ""some_table"."some_field_3" AS "field_3"",
            ...
            ""another_table"."some_field_N" AS "field_N"",
....
```

so, it's wrong.

problem here - https://github.com/75team/thinkjs/pull/451/files#diff-55a5d5e9ab2678195c04299e06527bdbR235

## ENV
dev/prod

Platform:

macOS / Linux / Win

Node.js Version:

v6.7.0

ThinkJS Version:

Release 2.2.12

**Comment by GuilhermeReda** [comment]
Fixed on pr #460

**Comment by im-kulikov** [comment]
fixed by #462
